include/utils_precindep.h

```diff
@@ -8,7 +8,9 @@
 
 BIGINT next235even(BIGINT n);
 
-// jfm's timer class
+namespace finufft{
+
+  // jfm's timer class
 #include <sys/time.h>
```

Comment:
Includes should be outside of namespace declarations, as they are textual (i.e. including is the same as copy pasting, so you would change the namespace of the declarations inside, which is problematic).